gst/tensor_aggregator/tensor_aggregator.c

```diff
@@ -525,13 +525,32 @@ gst_tensor_aggregator_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
       ret == GST_FLOW_OK) {
     GstBuffer *outbuf;
     GstClockTime pts, dts;
+    guint64 pts_dist, dts_dist;
     gsize flush, offset;
+    gint rate;
 
     /** offset for last frame */
     offset = frame_size * (frames_out - 1);
 
-    pts = gst_adapter_prev_pts_at_offset (adapter, offset, NULL);
-    dts = gst_adapter_prev_dts_at_offset (adapter, offset, NULL);
+    pts = gst_adapter_prev_pts_at_offset (adapter, offset, &pts_dist);
+    dts = gst_adapter_prev_dts_at_offset (adapter, offset, &dts_dist);
+
+    /**
+     * If frames-in is larger then frames-out, it will set the same timestamp.
+     * Update timestamp of last frame.
+     */
+    rate = self->in_config.rate_n / self->in_config.rate_d;
```

Comment:
Question: is it guaranteed to have ```self->in_config.rate_n % self->in_config.rate_d == 0```?

Reply:
If frames-in > 1 and framerate > 0, it can get new timestamp. 
Other cases, we cannot calculate the timestamp.

Reply:
I was asking what if n / d = 1.5.

Reply:
I have updated this code from value numerator and denominator.
Using gst_util_uint64_scale_int(), the result which is not exactly represented as integer will be truncated.